cmd/kubeadm/app/cmd/alpha/certs_test.go

```diff
@@ -30,9 +30,9 @@ import (
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	certsphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
 	kubeconfigphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubeconfig"
+	testutil "k8s.io/kubernetes/cmd/kubeadm/app/testing"
+	cmdtestutil "k8s.io/kubernetes/cmd/kubeadm/app/testing/cmd"
```

Comment:
so the introduction of cmd/kubeadm/app/testing is something good and i forgot about this problem where kubeadm imports it's integration tests for some utils.
i wasn't around to gate such a change, so it slipped trough in multiple packages...

so by moving such utils under /app, /app no longer has to import /test which is good.